README.md

```diff
@@ -44,16 +45,23 @@ jobs:
             github-token: ${{ secrets.TOKEN }}
 ```
 
+## Detailed Setup instructions
```

Comment:
As I have mentioned your video link in the featured content therefore there is no need for this file. You can remove this section.

Reply:
Ok as you say, I'll remove it.